resources/dicts/patrols/general/border.json

```diff
@@ -1569,5 +1569,388 @@
+                        "amount": 5
+                    }	
+                ]
+            }
+        ],
+        "fail_outcomes": [
+            {
+                "text": "The bickering gets to be too much, and r_c storms back to camp irritated.",
+                "exp": 0,
+                "weight": 20
+            },
+            {
+                "text": "While they are bickering, r_c trips, and p_l snickers, earning r_c's ire.",
+                "exp": 0,
+                "weight": 20,
+                "injury": {
+                    "cats": ["r_c"],
+                    "injuries": ["minor_injury"]
+                },
+                "relationships": [
+                    {
+                        "cats_to": ["p_l"],
+                        "cats_from": ["r_c"],
+                        "mutual": false,
+                        "values": ["dislike"],
+                        "amount": 3
+                    }	
+                ]
+            },
+            {
+                "text": "As the duo are patroling, they are ambushed by a rogue! r_c is badly injured, and succumbs to the wounds when p_l flees, leaving {PRONOUN/r_c/object} behind.",
+                "exp": 0,
+                "weight": 5,
+                "dead_cats": ["r_c"]
+            },
+            {
+                "text": "s_c can't resist the temptation to make pointed remarks {PRONOUN/s_c/subject} {VERB/s_c/know/knows} are hurtful to r_c. But when a verbal barb sinks too deep and r_c starts crying, s_c lashes their tail, snapping at r_c to focus on patrolling. r_c runs off, leaving s_c to finish the patrol alone.",
+                "exp": 0,
+                "weight": 10,
+                "can_have_stat": ["p_l"],
+                "stat_trait": ["troublesome", "fierce", "bloodthirsty", "cold", "childish", "shameless", "vengeful", "arrogant", "competitive", "grumpy", "cunning", "oblivous", "rebellious"],
+                "relationships": [
+                    {
+                        "cats_to": ["s_c"],
+                        "cats_from": ["r_c"],
+                        "mutual": true,
+                        "values": ["dislike"],
+                        "amount": 10
+                    },
+                    {
+                        "cats_to": ["p_l"],
+                        "cats_from": ["r_c"],
+                        "mutual": false,
+                        "values": ["platonic", "respect"],
+                        "amount": -5
+                    }		
+                ]
+            },
+            {
+                "text": "As s_c pounces on a bug, r_c can't help but snap that this is exactly what {PRONOUN/r_c/subject} dislikes about s_c. s_c retortes that r_c needs to learn to lighten up and enjoy life. r_c says that {PRONOUN/r_c/subject} {VERN/r_c/do/does} know how to have fun, and s_c says to prove it, but r_c says that {PRONOUN/r_c/subject} {VERB/r_c/don't/doesn't} have to prove anything to s_c.",
```

Comment:
for some reason it looks like the pronoun tags don't love {VERB/r_c/don't/doesn't} - change to {VERB/r_c/do/does}n't instead? I thin kit's the apostrophe